dynamodb/dynamodb-client.w

```diff
@@ -114,4 +114,32 @@ pub inflight class Client impl dynamodb_types.IClient {
     input.set("TableName", this.tableName);
     return unsafeCast(this.client.query(input));
   }
+
+  pub inflight batchGet(options: dynamodb_types.BatchGetOptions): dynamodb_types.BatchGetOutput {
+    return unsafeCast(this.client.batchGet(options));
+  }
+
+  pub inflight batchWrite(options: dynamodb_types.BatchWriteOptions): dynamodb_types.BatchWriteOutput {
+    let reqItems = MutMap<MutArray<dynamodb_types.WriteRequest>> {};
+
+    for item in options.RequestItems.entries() {
+      let tableReqs = MutArray<dynamodb_types.WriteRequest> [];
+
+      for DeleteRequest in item.value.DeleteRequests ?? [] {
+        tableReqs.push({ DeleteRequest });
+      }
+
+      for PutRequest in item.value.PutRequests ?? [] {
+        tableReqs.push({ PutRequest });
+      }
+
+      reqItems.set(item.key, tableReqs);
+    }
+
+    return unsafeCast(this.client.batchWrite({
+      RequestItems: Json.parse(Json.stringify(reqItems)),
```

Comment:
Sorry, new to Wing! There is probably a better way to get this thing into a Json type, I just couldn't seem to figure it out.

Reply:
One way to get mutable collections in Wing into Json is by obtaining immutable copies with the `.copy()` method, and then Wing lets you use many immutable arrays or collections in place of Json. But here it looks like the collection is "deep" so that might not work here. I think if we added some `toJson()` and `toMutJson()` methods to the language, perhaps that would be more ideal. I'll take a note and open a separate issue - thanks for the heads up.

Reply:
Good to know– that would be a useful set of functions to have!